src/experimental/dx-standalone-linux/test/smoke.sh

```
@@ -0,0 +1,74 @@
+# Env: DX_BIN (path to the dx-daemon binary). If unset, assumes dx already
+#      listens on :9099/:9095 (e.g. the systemd/compose deployment).
+set -uo pipefail
+HERE="$(cd "$(dirname "$0")" && pwd)"
+INJECT="$HERE/../inject/inject-referral.sh"
+DX_PORT=9099
+METRICS_PORT=9095
+LOG="$(mktemp)"
+PID=""
+
+# shellcheck disable=SC2317  # invoked via trap
+cleanup() { [ -n "$PID" ] && kill "$PID" 2>/dev/null; rm -f "$LOG"; }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)